src/main/java/com/verygood/security/coding/aliases/AliasCreateResponse.java

```diff
@@ -0,0 +1,12 @@
+package com.verygood.security.coding.aliases;
+
+import lombok.Data;
+
+@Data
+public class AliasCreateResponse {
+    private String alias;
+
+    public AliasCreateResponse(String alias) {
```

Comment:
Should be covered by `@RequiredArgsConstructor` which is a part of `@Data` annotation.
Reason why It's not there is the absence of `@NotNull` annotation.